### ISSUE TITLE
Lunar golems using helfire tamed by Chirr are OP

### ISSUE THREAD

**Issue by LordVGames** [FocusedFault/EnemiesPlus#2]
Playing as Chirr and getting a lunar golem to attack mithrix stacked a ton of helfire and it melted through his hp pretty quickly since helfire does damage based on hp %.

https://github.com/FocusedFault/EnemiesPlus/assets/51967559/83a04649-853e-4b90-b77f-c5e68d23e1c2

Could probably balance it when it's applied to an enemy it by either having it do a flat amount instead of a % of health or just having it not work on mithrix & maybe the other chimera or just on all enemies idk

**Comment by Nuxlar** [comment]
You're right i did not think about that, I could make it unstackable again or change how it works. Will have a fix out next update thanks for reporting!
